src/sourcemaps/sourcemaps-wizard.ts

```diff
@@ -73,37 +75,37 @@ You can turn this off by running the wizard with the '--disable-telemetry' flag.
 
   await traceStep('check-sdk-version', ensureMinimumSdkVersionIsInstalled);
 
-  const { url: sentryUrl, selfHosted } = await traceStep(
-    'ask-self-hosted',
-    () => askForSelfHosted(options.url),
-  );
-
-  const { projects, apiKeys } = await traceStep('login', () =>
-    askForWizardLogin({
```

Comment:
l: We have a bunch of unused imports because of the removals in this file.

Reply:
oof, that's what happens when you push a PR on Friday afternoon 🙃 Fixed (along with the other leftovers)